client/views/create/create.js

```diff
@@ -90,6 +90,9 @@ Template.create.events({
     const maxResponse = responseSelect[responseSelect.selectedIndex].value;
     const hiddenSelector = document.getElementsByName('visibility')[0];
     const isHidden = (hiddenSelector[hiddenSelector.selectedIndex].value === 'hidden');
+    const socialSelector = document.getElementsByName('social')[0]; // Extract the HTML select element for getting the social toggles
```

Comment:
@singhsanket143 can you move the two comments up?

Reply:
@peter-hank Updated the comments